### PR TITLE
added git and youtube icons @social links

### PR DESCRIPTION
**Pull request by saurabhlondhe**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -272,6 +272,12 @@ <h2 class="section-heading">Let's Get In Touch!</h2>
                 <div class="col-lg-4 col-md-4 col-sm-4 col-xs-4 text-left">
                     <a id="twitter" href="https://twitter.com/ritinlug" target="_blank"><i class="fa fa-twitter-square fa-3x sr-contact"></i></a>
                 </div>
+                 <div class="col-lg-4 col-lg-offset-2 col-md-4 col-md-offset-2 col-sm-4 col-sm-offset-2 col-xs-4 col-xs-offset-2 text-right">
+                    <a id="github" href="https://github.com/ritinlug" target="_blank"><i class="fa fa-github fa-3x sr-contact"></i></a>
+                </div>
+                 <div class="col-lg-4 col-md-4 col-sm-4 col-xs-4 text-left">
```
Comment (bhavin192) [inline index.html]: We don't have YouTube channel, please remove it at least for now

File: index.html [hunk 2]
```diff
@@ -272,6 +272,12 @@ <h2 class="section-heading">Let's Get In Touch!</h2>
                 <div class="col-lg-4 col-md-4 col-sm-4 col-xs-4 text-left">
                     <a id="twitter" href="https://twitter.com/ritinlug" target="_blank"><i class="fa fa-twitter-square fa-3x sr-contact"></i></a>
                 </div>
+                 <div class="col-lg-4 col-lg-offset-2 col-md-4 col-md-offset-2 col-sm-4 col-sm-offset-2 col-xs-4 col-xs-offset-2 text-right">
```
Comment (bhavin192) [inline index.html]: The color of the logo should match with other logo colors

**Review by bhavin192 — CHANGES_REQUESTED** [review]
Make it inline with other two logos, you can see the deployed preview here 
https://deploy-preview-10--ritinlug.netlify.com

**Review by bhavin192 — APPROVED** [review]

**Comment by shahajishinde** [comment]
LGTM 
The GitHub link, we missed it.

**Review by akshaygin — COMMENTED** [review]
- Make it inline with other logos
- github logo should square in shape, similar to other
- Make github logo responsive(It should have properly placed for different devices)

**Review by bhavin192 — CHANGES_REQUESTED** [review]
Please submit only one change per PR
Revert these changes of the README.md

**Review by bhavin192 — APPROVED** [review]

**Pull request merged**